masterlist.yaml

```
@@ -24908,3 +24908,11 @@ plugins:
   - name: 'MarkekrausExitCombatPower.esl'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/58651/' ]
     msg: [ *obsolete ]
+
+  - name: 'EdhildilsDMRAstore.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/39279/' ]
```

The location would benefit from a name:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/39279/'
        name: 'Edhildils Dmra Store'
```